src/sestest/GenerateCommandLine.cs

```diff
@@ -20,7 +20,7 @@ public sealed class GenerateCommandLine : CommandLineBase
         [Option("not-after", HelpText = "The moment at which the token expires and the application is no longer entitled to execute (format 'yyyy-mm-ddThh:mm'; 24 hour clock; local time; defaults to 7 days).")]
         public string NotAfter { get; set; }
 
-        [Option("address", HelpText = "The externally visible IP addresses of the machine entitled to execute the application(s) (defaults to the addresses of the current machine).")]
+        [Option("address", HelpText = "The externally visible IP addresses of the machine entitled to execute the application(s) (comma separated, defaults to the addresses of the current machine).", Separator = ',')]
```

Comment:
Is this a breaking change, or does this now support both space and comma separation?

[Concious that some of our partners may have test scripts reliant on using space as a separator; if we're going to change it, we need to call it out as a breaking change when we formally release.]

Reply:
I thought it would be breaking, but apparently it's not. Space still works as a separator. I wonder if we should have some unit tests around that, though? There's also some weirdness if you combine commas and spaces, as in `a, b, c`, in which the resulting collection contains empty strings.